source/includes/entityref/paybill/_billablecharge.md

```diff
@@ -322,6 +322,20 @@ Represents a billable charge.
             <td></td>
             <td>X</td>
         </tr>
+<tr class="odd">
+            <td>hasAdjustment</td>
+            <td>Boolean</td>
+            <td>Indicates whether billable charge has an adjustment.</td>
```

Comment:
You can add another helpful sentence about this field as well:

`This field is set automatically when an adjustment has occurred in the system, and it can also be manually overridden later if the charge needs to be manually adjusted.`